spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/subscriber/PubSubSubscriberOperations.java

```diff
@@ -38,14 +42,34 @@
  */
 public interface PubSubSubscriberOperations {
 
+	/**
+	 * @deprecated As of 1.1, use {@link #subscribe(String, Consumer)} instead.
```

Comment:
Huh? Deja Vu?

Seems for me we discussed with you that this should be fixed.

What am I missing?

Reply:
Oh! I know "why?". This PR is based on your in-progress one #923 

Can't we somehow do not combine work-in-progress in different PR?

Should I review the same code twice and comment about this "As" all the time ? 😉